met-web/src/components/dashboard/EngagementsAccordion.tsx

```diff
@@ -26,7 +26,7 @@ const EngagementsAccordion = ({
     const [openedEngagements, setOpenedEngagements] = useState<number[]>([]);
     const isMobile = useMediaQuery((theme: Theme) => theme.breakpoints.down('xs'));
     const [projectMapData, setProjectMapData] = React.useState<Map | null>(null);
-    const mapExists = projectMapData?.latitude && projectMapData?.longitude;
+    const mapExists = (projectMapData?.latitude && projectMapData?.longitude) ?? true;
```

Comment:
is this the result we're looking for here:
![image](https://github.com/bcgov/met-public/assets/91914654/2f3493cd-5e15-43fb-b17a-ec8fdbbd53f9)

Reply:
no at all